var/spack/repos/builtin/packages/catalyst/package.py

```diff
@@ -128,6 +130,13 @@ def setup_environment(self, spack_env, run_env):
                              paraview_version))
         run_env.prepend_path('LD_LIBRARY_PATH', join_path(lib_dir,
                              paraview_version))
+        if '+python' in self.spec:
+            run_env.prepend_path('PYTHONPATH', join_path(lib_dir,
+                                 paraview_version))
+            run_env.prepend_path('PYTHONPATH', join_path(lib_dir,
+                                 paraview_version, 'site-packages'))
+            run_env.prepend_path('PYTHONPATH', join_path(lib_dir,
+                                 paraview_version, 'site-packages', 'vtk'))
```

Comment:
Do all 3 of these really need to be in `PYTHONPATH`? It's usually just `lib/<whatever>/site-packages` that is necessary.

Reply:
you are right, it has been fixed in #8752 by @olesenm